S4TF_Tutorial_3.ipynb

```diff
@@ -1764,7 +1763,7 @@
         "Experiment with different models and see how the accuracy results differ. In particular change the following parameters:\n",
```

Comment:
Above this, can the plot warning be fixed, so that it doesn't show in the notebook during execution?